django/chat/models.py

```diff
@@ -331,6 +332,7 @@ class AnswerSource(models.Model):
     node_score = models.FloatField(default=0.0)
     # Saved citation for cases where the source Document is deleted later
     saved_citation = models.TextField(blank=True)
+    group_number = models.IntegerField(default=0)
```

Comment:
This is used to group sources when displaying them in the modal, for granular Q+A